js/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+window.error = (function () {
+  var errorTemplate = document.querySelector('#error').content.querySelector('.error');
+  var errorMessageElement = errorTemplate.cloneNode(true);
```

Comment:
line 4-6 should be inside errorHandler function. You create new template every time you generate an error screen

Reply:
done